general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -615,6 +615,13 @@ def _handle_status_codes(self, response: requests.Response) -> None:
             ):
                 self._prompt_accept_terms_of_use()
                 return
+
+            elif response.json() == ("You must validate your registered email."):
```

Comment:
Is this meant to be a tuple it's checking against? If not, I think the parentheses can be removed here.